src/state/mod.rs

```diff
@@ -66,6 +70,36 @@ impl event::EventHandler for MainState {
         graphics::present(ctx)?;
         Ok(())
     }
+
+    fn key_down_event(&mut self, _ctx: &mut Context, keycode: KeyCode, _keymods: KeyMods, _repeat: bool) {
+        match keycode {
+            KeyCode::W => {
+                self.player1.move_up();
+            },
+            KeyCode::S => {
+                self.player1.move_down();
+            },
+            KeyCode::Up => {
```

Comment:
Maybe `KeyCode::L` and `KeyCode::O` to keep the same experience as of `WS`

Reply:
Honestly I prefer `W` and `S` + `Up` and `Down` since that is what most games I've seen use, at least most of the flash ones hahahaha.

Reply:
OK